xc/device.ml

```diff
@@ -1755,8 +1782,9 @@ let __start (task: Xenops_task.t) ~xs ~dmpath ?(timeout = !Xenopsd.qemu_dm_ready
 			~name:"vgpu" ~domid ~xs ~ready_path ~timeout:!Xenopsd.vgpu_ready_timeout ~cancel () in
 		Forkhelpers.dontwaitpid vgpu_pid
 	end
-	| VNC (Vgpu _, _, _, _, _)
-	| SDL (Vgpu _, _) -> failwith "Unsupported vGPU configuration"
```

Comment:
Good catch - this is possible if you've somehow got zero or more than one vGPU. There shouldn't be any way of configuring this, but if it's happened we should fail rather than continuing.